### DISCUSSION TITLE
Question about Android Scene Viewer

### DISCUSSION THREAD

**Discussion by ranbuch** [google/model-viewer#3572] [Q&A]
I really want to apologize for opening this ticket here as this is not directly related to model-viewer but I just don't know who to ask.

I know there was a way to load multiple GLB files on scene viewer using some URL parameter that states the giles location and poster image. By any chance, does anyone know what are those parameters or can provide a link to an example? Or can refer me to the proper place to ask this question?

Thanks.

**Comment by elalish** [comment]
https://developers.google.com/ar/develop/scene-viewer#supported_intent_parameters

**Reply by ranbuch** [reply]
Thanks for the link. Unfortunately, although the option is available that's not in the support parameters list.

**Reply by elalish** [reply]
Hmm, if it's undocumented I'm not sure it's a stable API. @AdrianAtGoogle do you know more about this?
